src/build-task/AugurkCSharpAnalyzer/task.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "3b04573e-7699-440a-93ae-8f7b06259078",
+  "name": "AugurkCSharpAnalyzer",
+  "friendlyName": "Run Augurk C# Analysis",
+  "description": "Performs a static code analysis over your test automation and uploads the results to Augurk",
+  "category": "Utility",
+  "author": "Augurk",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 2
```

Comment:
This release isn't a patch but a minor version, right?

Reply:
We should version build tasks manually, since they need to be incremented every time in order to be able to test them.